go/pkg/tool/tool.go

```diff
@@ -326,6 +326,20 @@ func (c *Client) DownloadDirectory(ctx context.Context, rootDigest, path string)
 	return err
 }
 
+// UploadDirectory uploads a directory the specified path as a Merkle-tree to the remote cache.
+func (c *Client) UploadDirectory(ctx context.Context, path string) error {
+	log.Infof("Computing Merkle tree rooted at %s.", path)
```

Comment:
The period might look like a dot with the extension omitted.
```suggestion
	log.Infof("Computing Merkle tree rooted at %s", path)
```

Reply:
Done.